### DISCUSSION TITLE
How do I create a label with a color border but a white background color

### DISCUSSION THREAD

**Discussion by WForst-Breeze** [community/community#48619] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

just like this:
![image](https://user-images.githubusercontent.com/110760354/221395883-af37db24-510c-45ff-b46e-16ab9cb8522d.png)
Thanks for your help :D

**Comment by valentinojosh** [comment, marked as answer]
Hi,
I'm assuming you want this in HTML, CSS?

The following code should get you pretty close:

`<div class="oval-button">Label Text</div>`

`<style>`
`.oval-button {`
 ` background-color: white;`
 ` border: 2px solid blue;`
 ` border-radius: 20px; /* Adjust this value to change the roundness of the button */`
 ` padding: 10px; /* Optional: add padding to the inside of the button */`
 ` display: inline-block; /* Optional: display the button as an inline element */`
 ` text-align: center; /* Optional: center the text inside the button */`
`}`
`</style>`



Hope this helps!
